spyder/plugins/ipythonconsole/widgets/debugging.py

```diff
@@ -169,6 +193,8 @@ def __init__(self, *args, **kwargs):
         self._pdb_input_queue = []  # List of (code, hidden, echo_stack_entry)
         # Temporary flags
         self._tmp_reading = False
+        self._pdb_single_letter_mode = False
+        self._pdb_single_letter_prompt = "explore mode: "
```

Comment:
I don't like showing a different prompt for this. What about showing `IPdb [n]*:` to signal that something changed in the way we're evaluating things?

Reply:
Well it is a pretty fundamental change, We are only accepting a single letter that is executed without the need of pressing enter. I don't think this should look like a prompt really. I was going for something like a vim command mode